addons/website_blog/views/website_blog_components.xml

```diff
@@ -46,7 +46,7 @@
                     <t t-esc="tag.name"/>
                     <a t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, tag.id))}" class="btn border-0 py-1 post_link" t-att-rel="len(active_tag_ids) and 'nofollow'">&#215;</a>
                 </span>
-                <a t-elif="showInactive" t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, tag.id))}" t-attf-class="badge mb-2 mw-100 text-truncate #{tag.id in active_tag_ids and 'text-bg-primary' or 'border'} post_link" t-attf-rel="len(active_tag_ids) and 'nofollow'" t-esc="tag.name"/>
+                <a t-elif="showInactive" t-attf-href="#{blog_url(tag=tags_list(active_tag_ids, tag.id))}" t-attf-class="badge mb-2 mw-100 text-truncate text-decoration-none #{tag.id in active_tag_ids and 'text-bg-primary' or 'border text-primary'} post_link" t-attf-rel="len(active_tag_ids) and 'nofollow'" t-esc="tag.name"/>
```

Comment:
I get the text-primary, otherwise the text is white on white.
But why do you need the `text-decoration-none`?

Note that in commit message you wrote `text-no-decoration`: `This commit fixes that by adding .text-primary and .text-no-decoration to the badge.`

Reply:
I changed the commit message, thanks for pointing it out 👍🏻 

For the text-decoration-none, it's just to restore the style prior to the Bootstrap 5 migration, without it, hovering the link displays an underline. (Which is not the case in <=15.3)

Here I removed the class from the adventure tag:
![CleanShot 2022-10-26 at 13 12 44](https://user-images.githubusercontent.com/77747125/198012462-0149223d-5ab8-4c1c-8b53-f91e9ea634ab.gif)